cordlib/src/main/kotlin/com.luxoft.blockchainlab.corda.hyperledger.indy/flow/b2b/IssueCredentialFlowB2B.kt

```diff
@@ -147,8 +146,7 @@ object IssueCredentialFlowB2B {
                 val offer = flowSession.receive<CredentialOffer>().unwrap { offer -> offer }
                 val sessionDid = subFlow(CreatePairwiseFlowB2B.Prover(issuer))
 
-                val credentialRequestInfo =
-                    indyUser().createCredentialRequest(sessionDid, offer, indyUser().defaultMasterSecretId)
+                val credentialRequestInfo = indyUser().createCredentialRequest(sessionDid, offer)
```

Comment:
What is happened with masterSecret?

Reply:
masterSecretId is an identifier of some cryptographic tool used by prover to create ZKP. It's moved to IndyUser constructor (where it can be configured) and hidden in other places.

Also, `defaultMasterSecretId` was deprecated from "~0.4.10"

Reply:
Well, there is tricky question. Master user is used to connect different credentials from cryptography side. It ideal world it should be dictated by end-user and come from outside. In company case it actually comes from flow, on top layer we can manage it and get from config or special storage.